tests/framework/common.go

```diff
@@ -1239,7 +1240,13 @@ func (td *OsmTestData) Cleanup(ct CleanupType) {
 				},
 			)
 			if err != nil {
-				td.T.Logf("Poll err: %v", err)
+				td.T.Logf("Cleanup of namespaces timed out: %s", err)
```

Comment:
This error could happen for reasons other than just timeouts. If it does time out, the error will say that, so I'd make this message more generic just to avoid confusion if the error isn't really a timeout.

Reply:
Updated.